apps/remote_control/lib/lexical/remote_control/build/error.ex

```diff
@@ -49,21 +49,36 @@ defmodule Lexical.RemoteControl.Build.Error do
     )
   end
 
-  defp uniq(diagnostics) do
+  def uniq(diagnostics) do
     # We need to uniq by position because the same position can be reported
     # and the `end_line_diagnostic` is always the precise one
     extract_line = fn
       %Result{position: {line, _column}} -> line
       %Result{position: {start_line, _start_col, _end_line, _end_col}} -> start_line
-      %Result{position: line} when is_integer(line) -> line
+      %Result{position: line} -> line
```

Comment:
we can't always find the integer line.

for example: 

```elixir
        ~S[
          Module.create(
            Foo,
            quote do
              String.to_integer("")
            end,
            file: "")
      ]
```

Do you think it is better to crash here rather than the diagnostics genserver?

if so, I can change this test: `    test "can't find right line when use macro" do`